### PR TITLE
[ADHOC] Minor Updates for ERC20PeggedIncentive

### PR DESCRIPTION
**Pull request by jonathandiep**
### Description
- Update some comments
- Added `peg` function to read the state variable
- Deployed `ERC20PeggedIncentive` to Sepolia

### CODE REVIEW

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Your PR body must reference a Github issue using a valid keyword, or your PR title must include the internal Boost ticket number, ie: `[BOOST-1234] Innovate like nuts`
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Are you sure you want to be submitting a change without including a changeset? If you're just changing docs or tests, you probably don't need to. See [the publishing section of the README](https://github.com/boostxyz/boost-protocol?tab=readme-ov-file#changesets--publishing) for more info.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 033270db1a02e3645b6272a3f99ef3daa76efde7
</p>

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Your PR body must reference a Github issue using a valid keyword, or your PR title must include the internal Boost ticket number, ie: `[BOOST-1234] Innovate like nuts`
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8dd39d5d09f7f0855624dfcb9dc7cff9bfc604d2
</p>

File: packages/sdk/src/Incentives/ERC20PeggedIncentive.ts [hunk 1]
```diff
@@ -231,6 +232,22 @@ export class ERC20PeggedIncentive extends DeployableTarget<
...
 
+  /**
+   * The address of the pegged ERC20-like token
+   *
+   * @public
+   * @async
+   * @param {?ReadParams} [params]
+   * @returns {Promise<Address>}
+   */
+  public async peg(params?: ReadParams) {
```
Comment (Quazia) [inline packages/sdk/src/Incentives/ERC20PeggedIncentive.ts:243]: [I think this covers the need already](https://github.com/boostxyz/boost-protocol/blob/57f089f706a0dc52b3e26de3177068c3a05b20f1/packages/sdk/src/Incentives/ERC20PeggedIncentive.ts#L366)

**Review by Quazia — CHANGES_REQUESTED** [review]

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  By using the `[ADHOC]` title prefix, you are bypassing best practice protections.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 25dac1859d07d32483a21d7ddb24180c20916396
</p>

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  By using the `[ADHOC]` title prefix, you are bypassing best practice protections.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cebecf874ca448f74463fbdba35b4ab79b7ef1cc
</p>

**Review by Quazia — APPROVED** [review]
LG - thanks for the comment cleanup 😅

**Pull request merged**